### PR TITLE
Moving oidc_config to new framework

### PR DESCRIPTION
**Pull request by tsorya**
Moving oidc_config to new framework

subsystem passes

### CODE REVIEW

File: provider/groupmembership/group_membership_state.go
Comment (osherdp) [inline provider/groupmembership/group_membership_state.go:1]: Can you clarify what happens here? Are we deleting oidc-config resource and github mistakenly considers it a rename to group-membership resource?

Comment (tsorya) [reply provider/groupmembership/group_membership_state.go:1]: It is mess , sec i am fixing it

Comment (tsorya) [reply provider/groupmembership/group_membership_state.go:1]: fixed

**Pull request merged**